docusaurus/docs/Android/02-client/06-guides/07-chatdomain-migration.mdx

```diff
@@ -0,0 +1,167 @@
+# ChatDomain Migration
+
+**v5.0.0** release brings a big change to the offline support library - it replaces `ChatDomain` with a new, easy-to-use `OfflinePlugin`.
+You can read more about the motivation behind the effort and featured changes in the [announcement blog post](https://getstream.io/blog/).  <!--TODO: Replace with a real link -->
```

Comment:
The permalink might be `https://getstream.io/blog/stream-chat-android-v5-sdk-release/`. I will keep you updated on this.